src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,28 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo | null,
```

Comment:
You render this component only if you have `selectedTodo`, so to can't be `null`
```suggestion
  todo: Todo,
```